changelog.md

```diff
@@ -1,3 +1,10 @@
+### <a name="2.14.0"></a> 2.14.0
+
+* Add PopulateIndexMetrics to FeedOptions
```

Comment:
@timsander1 can you please provide a one sentence explaining the feature.

Reply:
Let use: Added an option to include index metrics in the response.

Reply:
Can we add more details? Add PopulateIndexMetrics to FeedOptions which allows users to get index usage during testing to check the efficiency of the query.

Reply:
Sure. We can use: Add PopulateIndexMetrics to FeedOptions which allows users to get index usage metrics during testing to improve query performance..